frontend/src/app/projects/models/model-detail/t4c-model-wrapper/t4c-model-wrapper.component.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { map, combineLatest, switchMap, filter } from 'rxjs';
+import { ModelService } from 'src/app/services/model/model.service';
+import { ProjectService } from 'src/app/services/project/project.service';
+import { T4cModelService } from 'src/app/services/source/t4c-model.service';
+
+@Component({
+  selector: 'app-t4c-model-wrapper',
+  templateUrl: './t4c-model-wrapper.component.html',
+  styleUrls: ['./t4c-model-wrapper.component.css'],
+})
+export class T4cModelWrapperComponent implements OnInit, OnDestroy {
+  constructor(
+    private route: ActivatedRoute,
+    public projectService: ProjectService,
+    public modelService: ModelService,
+    private t4cModelService: T4cModelService
+  ) {}
+
+  ngOnInit(): void {
+    combineLatest([
+      this.projectService._project.pipe(
+        filter(Boolean),
+        map((p) => p.slug)
```

Comment:
```suggestion
        map((project) => project.slug)
```
As mentioned above, I personally prefer descriptive names rather than using short names.